database/migrations/2023_07_06_101552_add_timestamps_to_course_group_pivot_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddTimestampsToCourseGroupPivotTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('course_group', function (Blueprint $table) {
+            $table->timestamps();
```

Comment:
please check if it will work for existing entries, or if you need to add nullable()
```suggestion
            $table->timestamps()->nullable();
```

Reply:
`timestamps()` is void so `nullable()` cant be applied, i think its nullable by default, like in course_user, before i added `withTimestamps()` to Course Users relationship, all timestamps were null